sdk/python/kfp/_client.py

```diff
@@ -688,6 +688,18 @@ def upload_pipeline(self, pipeline_package_path, pipeline_name=None):
       IPython.display.display(IPython.display.HTML(html))
     return response
 
+  def upload_pipeline_version(self, pipeline_package_path, version_name, pipeline_id):
```

Comment:
Could you use `client.pipeline_uploads.upload_pipeline_version()` instead of creating a helper?